ffi/dotnet/Devolutions.IronRdp/src/Framed.cs

```diff
@@ -20,19 +21,21 @@ public Framed(S stream)
 
     public async Task<(Devolutions.IronRdp.Action, byte[])> ReadPdu()
     {
-
         while (true)
         {
+            Console.WriteLine("ReadPdu, buffer size: " + this.buffer.Count);
             var pduInfo = IronRdpPdu.New().FindSize(this.buffer.ToArray());
             if (null != pduInfo)
             {
+                Console.WriteLine("ReadPdu: ReadExact, pduInfo.GetLength()=" + pduInfo.GetLength());
                 var frame = await this.ReadExact(pduInfo.GetLength());
                 var action = pduInfo.GetAction();
                 return (action, frame);
             }
             else
             {
                 var len = await this.Read();
+                Console.WriteLine("ReadPdu: Read, len=" + len);
 
```

Comment:
issue: The output may be too verbose for a basic terminal output. This is also part of the library, so all of this should be removed.

Reply:
updated